pkg/action/action_test.go

```diff
@@ -173,7 +173,17 @@ func withName(name string) chartOption {
 }
 
 func withSampleValues() chartOption {
-	values := map[string]interface{}{"someKey": "someValue"}
+	values := map[string]interface{}{
+		"someKey": "someValue",
+		"nestedKey": map[string]interface{}{
+			"simpleKey": "simpleValue",
+			"anotherNestedKey": map[string]interface{}{
+				"yetAnotherNestedKey": map[string]interface{}{
+					"youReadyForAnotherNestedKey?": "No",
```

Comment:
Might want to remove the `?` just in case here. If it goes through a yaml marshaler in a future test, I am not sure what will happen

Reply:
Sure! I'll do that. Also, about the third part package, is it okay to put the code at that level or should there be something like a "util" package inside the third party directory?

Reply:
I have made the changes for the `?`